_events/2022-0124.markdown

```diff
@@ -1,18 +1,18 @@
 ---
 
-eventdate: 2022-01-18T07:00
-tz: UTC -7
-title: OpenSearch Community Meeting - 2022-01-18
+eventdate: 2022-01-24T09:00
+tz: UTC -8
+title: OpenSearch Community Meeting - 2022-01-24 - open office hour
 online: true
 signup:
-    url: https://www.meetup.com/OpenSearch/events/282613028/
+    url: https://www.meetup.com/OpenSearch/events/282613041/
     title: Join on Meetup
 
 ---
 
```

Comment:
Maybe we should include that has been moved to accommodate MLK?

Reply:
Edit complete